web/src/compose_reply.js

```diff
@@ -118,6 +131,7 @@ export function respond_to_message(opts) {
         private_message_recipient: pm_recipient,
         trigger: opts.trigger,
         is_reply: true,
+        keep_composebox_empty: opts.keep_composebox_empty,
     });
 }
 
```

Comment:
I think the quote-and-reply function is missing this?

``` diff
diff --git a/web/src/compose_reply.js b/web/src/compose_reply.js
index f8e68ccef1..da1d0d9c51 100644
--- a/web/src/compose_reply.js
+++ b/web/src/compose_reply.js
@@ -163,7 +163,7 @@ export function quote_and_reply(opts) {
         // are prone to glitches where you select the
         // text, plus it's a complicated codepath that
         // can have other unintended consequences.)
-        respond_to_message(opts);
+        respond_to_message({...opts, keep_composebox_empty: true});
     }
 
     compose_ui.insert_syntax_and_focus(quoting_placeholder, $textarea, "block");
```

Reply:
Oh shoot yes, that one's important! Thanks.

It was also missing from `reply_with_mention` so I added it to that as well.

I think I was grepping for `.respond_to_message` so I missed the calls within the module `compose_reply.js` but that should be all fixed now.

Reply:
Makes sense. I really wish there was a way to make `git grep` find those local calls too, but you really need something fancier for it.